packages/mui-system/src/cssVars/createCssVarsProvider.d.ts

```diff
@@ -69,6 +75,12 @@ export default function createCssVarsProvider<
    * @default ''
    */
   prefix?: string;
+  /**
+   * A function to determine if the key, value should be attached as CSS Variable
+   * `keys` is an array that represents the nested object path.
+   *  Ex, { foo: { bar: 'var(--test)' } } => `keys`: ['foo', 'bar'], value: 'var(--test)'
+   */
+  shouldSkipVar?: (keys: string[], value: string | number) => boolean;
```

Comment:
> `keys` is an array that represents the nested object path.

Maybe we can come up with a clearer name than `keys`? e.g., `pathKeys` or `objectPathKeys`?

Reply:
`objectPathKeys` sounds better.